include/zephyr/usb/class/midi.h

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright (c) 2024 Titouan Christophe
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_INCLUDE_USB_CLASS_MIDI_H_
+#define ZEPHYR_INCLUDE_USB_CLASS_MIDI_H_
+
```

Comment:
Could you please add the 
```
#ifdef __cplusplus
extern "C" {
#endif

```with corresponding
```
#ifdef __cplusplus
}
#endif
```
sections to the header to suppress name mangling in C++? Like in "hid.h"...